ERCS/erc-7092.md

```diff
@@ -403,9 +508,15 @@ contract ERC7092Convertible is ERC7092 {
 }
 ```
 
+### Identity Registry
+
+This standard is designed specifically for tokenizing bonds. It does not inherently manage information pertaining to bondholders' identities. However, to enhance compliance with regulatory requirements and improve transparency, it is strongly RECOMMENDED to implement an identity registry to store the identity addresses of all authorized investors.
```

Comment:
Similarly, no requirements in reference implementation.

Reply:
> Similarly, no requirements in reference implementation.

Thank you @SamWilsn .

I just fixed it.